bundles/org.openhab.binding.hdpowerview/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -47,16 +55,28 @@
 	<channel-type id="scene-activate">
 		<item-type>Switch</item-type>
 		<label>Activate</label>
+		<tags>
+			<tag>Switch</tag>
+			<tag>Mode</tag>
+		</tags>
 	</channel-type>
 
 	<channel-type id="scene-group-activate">
 		<item-type>Switch</item-type>
 		<label>Activate</label>
+		<tags>
+			<tag>Switch</tag>
+			<tag>Mode</tag>
```

Comment:
Same here, just reminder to change here as well, if we will end up agreeing to a change.